### PR TITLE
Update dotnet docs logic in tools repo

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update dotnet docs logic in tools repo to prevent overwrite when sync happens.

### CODE REVIEW

File: eng/common/scripts/copy-docs-to-blobstorage.ps1 [hunk 1]
```diff
@@ -257,30 +257,31 @@ if ($Language -eq "javascript")
...
+    New-Item -ItemType directory -Path $DocsStagingDir
+    New-Item -ItemType directory -Path $TempDir
+
+    Expand-Archive -LiteralPath $PublishedDocs[0].FullName -DestinationPath $DocsStagingDir
+    $pkgProperties = ParseNugetPackage -pkg $PublishedPkgs[0].FullName -workingDirectory $TempDir
+
+    Write-Host "Start Upload for $($pkgProperties.ReleaseTag)"
+    Write-Host "DocDir $($DocsStagingDir)"
+    Write-Host "PkgName $($pkgProperties.PackageId)"
+    Write-Host "DocVersion $($pkgProperties.PackageVersion)"
```
Comment (sima-zhu) [inline eng/common/scripts/copy-docs-to-blobstorage.ps1:283]: Could you add a printout for release tag as well?

**Review by sima-zhu — COMMENTED** [review]
Overall, looks good to me. Could you please attach the template test? Did not see we successfully print out the release tag last time

File: eng/common/scripts/copy-docs-to-blobstorage.ps1 (hunk 1 above)
Comment (sima-zhu) [reply eng/common/scripts/copy-docs-to-blobstorage.ps1:283]: Missed the release tag printout. Ignore the comments

Comment (chidozieononiwu) [reply eng/common/scripts/copy-docs-to-blobstorage.ps1:283]: Its printed out in the Upload to Blob function.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**